src/server/hset_family.cc

```diff
@@ -518,12 +521,12 @@ OpResult<string> OpGet(const OpArgs& op_args, string_view key, string_view field
 
   DCHECK_EQ(pv.Encoding(), kEncodingStrMap2);
   StringMap* sm = GetStringMap(pv, op_args.db_cntx);
-  sds val = sm->Find(field);
+  auto it = sm->Find(field);
 
-  if (!val)
+  if (it != sm->end())
     return OpStatus::KEY_NOTFOUND;
```

Comment:
Here's the bug that causes hset test failures : it should be `it == sm->end()`

Reply:
thanks